### PR TITLE
PLT-47 Add tutorial for using tx constraint API

### PR DESCRIPTION
**Pull request by koslambrou**
Pre-submit checklist:
- Branch
    - [x] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Formatting, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested

### CODE REVIEW

**Comment by andreabedini** [comment]
@koslambrou is this still a draft or it is finished?

**Comment by koslambrou** [comment]
Waiting review by @joseph-fajen

File: doc/plutus/tutorials/basic-apps-constraints.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.. highlight:: haskell
+.. _basic_apps_constraints_tutorial:
+
+Extending the basic Plutus app with the constraints API
+=======================================================
+
+The previous tutorial (see :ref:`basic_apps_tutorial`) showed how you to write a Plutus app that locks some Ada in a script output and splits them evenly between two recipients.
```
Comment (joseph-fajen) [inline doc/plutus/tutorials/basic-apps-constraints.rst:7]: Please change "showed how you to write" to: 

"showed you how to write"

Comment (koslambrou) [inline doc/plutus/tutorials/basic-apps-constraints.rst:7]:
```suggestion
The previous tutorial (see :ref:`basic_apps_tutorial`) showed you how to write a Plutus app that locks some Ada in a script output and splits them evenly between two recipients.
```

**Review by joseph-fajen — APPROVED** [review]

**Pull request merged**